README.md

```diff
@@ -194,6 +194,52 @@ In other words, the new flag would indicate several connected changes related to
 
 For more discussion see [issue 2](https://github.com/tc39/proposal-regexp-set-notation/issues/2).
 
+### How is the `v` flag different from the `u` flag?
+
+The answer to this question can be useful when “upgrading” existing `u` RegExps to use `v`. Here’s an overview of the differences:
+
+1. (This is the obvious part.) Previously invalid patterns making use of the new syntax (see above) now become valid, e.g.
+
+    ```
+    [\p{ASCII_Hex_Digit}--[Ff]]
+    \p{RGI_Emoji}
+    [_\q{a|bc|def}]
+    ```
+
+1. Some previously valid patterns are now errors, specifically those with a character class including either an unescaped [special character](https://arai-a.github.io/ecma262-compare/snapshot.html?pr=2418#prod-ClassSetSyntaxCharacter) `(` `)` `[` `]` `{` `}` `/` `-` `\` `|` or [a double punctuator](https://arai-a.github.io/ecma262-compare/snapshot.html?pr=2418#prod-ClassSetReservedDoublePunctuator):
```

Comment:
```suggestion
1. Some previously valid patterns are now errors, specifically those with a character class including either an unescaped [special character](https://arai-a.github.io/ecma262-compare/snapshot.html?pr=2418#prod-ClassSetSyntaxCharacter) `(` `)` `[` `{` `}` `/` `|` (note: `\` and `]` and `-` are omitted from this list since they require escaping with the `u` flag already) or [a double punctuator](https://arai-a.github.io/ecma262-compare/snapshot.html?pr=2418#prod-ClassSetReservedDoublePunctuator):
```